pkg/test/resourcefixture/testdata/basic/bigquery/v1beta1/bigquerydatasetaccessblock/create.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Google LLC
```

Comment:
Could you re-structure the test cases based on our [guide](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#test-multiple-scenarios-for-the-same-resource-kind)? The expected folder structure will be:

```
pkg/test/resourcefixture/testdata/basic
├── bigquery
|   ├── v1beta1
|       ├── bigquerydataset
|         ├── basicbigquerydataset
|             └── create.yaml
|             └── update.yaml
|         ├── bigquerydatasetaccessblock
|             └── create.yaml
|             └── update.yaml
```

Reply:
Done